contrib/kafka/config/500-controller.yaml

```diff
@@ -32,10 +32,12 @@ spec:
       containers:
       - name: manager
         image: github.com/knative/eventing-sources/contrib/kafka/cmd/controller
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
I could remove it instead of making IfNotPresent as I think k8s default policy is Always - do you want ot to be removed? @matzew @dubee @lionelvillard

Reply:
This will break integration tests